src/eva/vision/callbacks/loggers/batch/segmentation.py

```diff
@@ -106,34 +107,52 @@ def _subsample_tensors(
     return tensors_stack
 
 
-def _denormalize_image(
-    tensor: torch.Tensor,
+def _descale_image(
+    image: torch.Tensor,
     mean: Iterable[float] = (0.0, 0.0, 0.0),
     std: Iterable[float] = (1.0, 1.0, 1.0),
     inplace: bool = True,
 ) -> torch.Tensor:
     """De-normalizes an image tensor to (0., 1.) range.
 
     Args:
-        tensor: An image float tensor.
+        image: An image float tensor.
         mean: The normalized channels mean values.
         std: The normalized channels std values.
         inplace: Whether to perform the operation in-place.
-            Defaults to `True`.
 
     Returns:
         The de-normalized image tensor of range (0., 1.).
     """
     if not inplace:
-        tensor = tensor.clone()
+        image = image.clone()
 
     return functional.normalize(
-        tensor,
+        image,
         mean=[-cmean / cstd for cmean, cstd in zip(mean, std, strict=False)],
         std=[1 / cstd for cstd in std],
     )
 
 
+def _denorm_image(image: torch.Tensor, inplace: bool = True) -> torch.Tensor:
```

Comment:
Should we move this to a public vision utils module. There might be other usecases for such functions coming up.